components/wrapper/Wrapper.jsx

```diff
@@ -0,0 +1,9 @@
+const Wrapper = ({children}) => {
+    return ( 
+       <section className="xl:mx-auto xl:max-w-320 w-320 xl:px-0 lg:px-9.5 px-5 max-w-97.5 md:max-w-233 mx-auto">
+        {children}
+       </section>
+     );
+}
```

Comment:
It would be better to name a wrapper component according to what is doing, like `PageWrapper` or `SectionWrapper` because it would be possible to have different wrappers in the same project.

Also, the folder should be named `wrappers` as it would contain many wrappers